### DISCUSSION TITLE
Different DNS Upstream Server listed in Dashboard

### DISCUSSION THREAD

**Discussion by cust0muser** [AdguardTeam/AdGuardHome#6262] [Q&A]
Hey, hope you are all doing well. I have the problem that DNS upstream servers are displayed in the Adguard dashboard that I have not entered in the settings.  How come?

![Screenshot_20230927_120619_DuckDuckGo.jpg](https://github.com/AdguardTeam/AdGuardHome/assets/125823408/9bf8883b-98b5-4271-a7b8-aa14daf171d3)

![Screenshot_20230927_120710_DuckDuckGo.jpg](https://github.com/AdguardTeam/AdGuardHome/assets/125823408/f3dfa010-500c-4cff-9f7d-82c1a1d9677c)

My upstream server ist just 192.168.112.5. Why is 192.168.114.1:53 listed? 
Where does this entry come from?

**Comment by ainar-g** [comment]
If you haven't configured any custom upstreams for clients, it is likely that it's the system default used for the private rDNS queries.  Check the *Private reverse DNS servers* area on the *Settings → DNS settings* page.

**Reply by cust0muser** [reply]
Yes, in fact, in the settings under private reverse DNS servers I have the addresses 
192.168.112.1, 
192.168.113.1 and 
192.168.114.1.  But they're just for name resolution, aren't they?

**Reply by ainar-g** [reply]
Are you implying that they shouldn't be counted then?  Because they are still upstream DNS servers that are still used by AdGuard Home.

**Reply by cust0muser** [reply]
Yes, that's what I wanted to say.  All DNS requests should go to 192.168.112.5.  The others should only be used to resolve IP addresses.  Have I set this up incorrectly?

**Reply by ainar-g** [reply]
Well, rDNS is a part of DNS.  rDNS queries, the ones used to get a domain name from an IP, are just another kind of DNS queries.  Sorry, I just really don't get your point here.
